src/components/banners/alert/alert.component.ts

```diff
@@ -12,9 +18,11 @@ export class AlertComponent {
   public static readonly TYPE_WARNING = 'warning';
   public static readonly TYPE_CONFIRMATION = 'confirmation';
   public static readonly TYPE_SUCCESS = 'success';
+  public static readonly TYPE_ERROR = 'error';
 
   @Input()
-  public type: 'warning' | 'confirmation' | 'success' = AlertComponent.TYPE_WARNING;
+  public type: AlertMessageType;
```

Comment:
Has the 'confirmation' type disappeared? Was that intentional?

Reply:
I'll get rid of this. The types were already in there so it should be dead code now really

Reply:
Need confirmation on what to do about alert-icon-class-pipe.ts regarding this